src/main/java/org/sagebionetworks/web/client/widget/evaluation/EvaluationRowWidget.java

```diff
@@ -75,10 +94,11 @@
 	}
 
 	@Inject
-	public EvaluationRowWidget(UserBadge userBadge, DateTimeUtils dateTimeUtils, SubmitToEvaluationWidget submitToEvaluationButton) {
+	public EvaluationRowWidget(UserBadge userBadge, DateTimeUtils dateTimeUtils, SubmitToEvaluationWidget submitToEvaluationButton, SynapseJavascriptClient synapseClient) {
 		this.userBadge = userBadge;
 		this.dateTimeUtils = dateTimeUtils;
 		this.submitToEvaluationButton = submitToEvaluationButton;
+		this.synapseClient = synapseClient;
```

Comment:
This is usually named "jsClient" in other areas of the code (different from the old GWT SynapseClient servlet RPCs)